Adding docs for Migration Readiness Report (#507)
neo4j/docs-aura: modules/ROOT/pages/auradb/managing-databases/migration-readiness.adoc

* Driver that was used to execute the query
* The application name that executed the query (if set)
* The initiation type of the query
* The query text or parts of it

Clicking on the "Fetch" button in the popup will populate the report's table with the applicable data.

image::mrr-deprecation-table.png[]

Each row in the table represents a query in the selected timeframe that must be changed to seamlessly migrate to the latest Aura version.
All executions of the same query are aggregated into one row (see also the "Count" column).
At the start of each row, clicking on the magnifying glass gives you a popup with more information about the query and suggestions on dealing with each issue.
It also provides relevant links to the documentation for each deprecation.

image::mrr-resolution-guide.png[width=600]

The last column in the table of Cypher deprecations links to a view of this specific query in the Aura Query Log Analyzer tool, which can provide information on each execution of the selected query.
The tool can only view queries on non-system databases.

image::mrr-show-query-log-button.png[width=400]

== Deprecated driver usage

image::mrr-fetch-driver-stats.png[width=400]

> After selecting a time frame of at most 24 hours, you can click the "Fetch driver statistics" button in this section.

Author: You sure about the end with ``` ?

> After selecting a time frame of at most 24 hours, you can click the "Fetch driver statistics" button in this section.

Reviewer: No idea where those came from!

By default, the filters in the popup are set to show only driver usage with potential issues in any database, including the system database.

> You can change those freely to see all driver usage, e.g.

Reviewer: ```suggestion
You can change those freely to see all driver usage, for example.
```

> Clicking on the "Fetch" button in the popup will populate the report's table with the applicable data.

Reviewer: ```suggestion
Use the button in the popup to fetch applicable data to populate the report's table.
```

Depending on the type of client accessing the Neo4j database, links are provided in the column “Latest version” to help with the upgrade.

image::mrr-driver-table.png[]

Like the Cypher deprecations table, the last column links to a view of this specific driver's executed queries in the Aura Query Log tool.
The tool can provide information on each query execution in which the selected driver was used.

> The tool can only view queries on non-system databases.

Reviewer: ```suggestion
The tool can view queries on all databases except the `system` database.
```

=== Deprecated index types

This section provides information on how to deal with deprecated indexes that may be used in version 4 but need to be handled before or while moving to version 5.

This part involves manually running a provided Cypher query on your database to identify the deprecated indexes and then deciding how to best deal with them.
Further enhancements to this feature will be provided in the future.

=== Next steps

After implementing all the recommended fixes from the report, you can now test the migration.

> To do that, please follow the steps outlined in the docs after pressing the "Test migration" button at the bottom of the page.

Reviewer: ```suggestion
To do that, follow the steps outlined in the docs after pressing the "Test migration" button at the bottom of the page.
```
Which docs are you referring to?

Author: The docs that are linked directly from the MRR page. So far its this link: https://neo4j.com/docs/aura/tutorials/upgrade/#_perform_the_upgrade

Reviewer: Alright, in that case, I suggest:
_Use the "Test migration" button at the bottom of the page and then follow the steps outlined in the docs._

* Driver that was used to execute the query
* The application name that executed the query (if set)
* The initiation type of the query
* The query text or parts of it

Use the button in the popup to fetch applicable data to populate the report's table.

image::mrr-deprecation-table.png[]

Each row in the table represents a query in the selected timeframe that must be changed to seamlessly migrate to the latest Aura version.
All executions of the same query are aggregated into one row (see also the "Count" column).
Use the magnifying glass at the start of each row to access a popup with more information about the query and suggestions on dealing with each issue.
It also provides relevant links to the documentation for each deprecation.

image::mrr-resolution-guide.png[width=600]

The last column in the table of Cypher deprecations links to a view of this specific query in the Aura Query Log Analyzer tool, which can provide information on each execution of the selected query.
The tool can view queries on all databases except the `system` database.

image::mrr-show-query-log-button.png[width=400]

== Deprecated driver usage

image::mrr-fetch-driver-stats.png[width=400]

> After selecting a time frame of at most 24 hours, you can click the "Fetch driver statistics" button in this section.

Author: ```suggestion
After selecting a time frame of a maximum of 24 hours, use the button to fetch the driver statistics in this section.
```

You can change those freely to see all driver usage, for example.

Use the button in the popup to fetch applicable data to populate the report's table.
Depending on the type of client accessing the Neo4j database, links are provided in the column “Latest version” to help with the upgrade.

image::mrr-driver-table.png[]

Like the Cypher deprecations table, the last column links to a view of this specific driver's executed queries in the Aura Query Log tool.
The tool can provide information on each query execution in which the selected driver was used.
The tool can view queries on all databases except the `system` database.

=== Deprecated index types

This section provides information on how to deal with deprecated indexes that may be used in version 4 but need to be handled before or while moving to version 5.

This part involves manually running a provided Cypher query on your database to identify the deprecated indexes and then deciding how to best deal with them.
Further enhancements to this feature will be provided in the future.

=== Next steps

After implementing all the recommended fixes from the report, you can now test the migration.

> To do that, please follow the steps outlined in the docs after pressing the "Test migration" button at the bottom of the page.

Author: ```suggestion
Use the "Test migration" button at the bottom of the page and then follow the steps outlined in the docs.
```
